contracts/contracts/OneTimeSBT.sol

```diff
@@ -100,62 +100,16 @@ contract OneTimeSBT is ERC721Enumerable {
         attributes.values[4] = AttributeLibrary.getDateOfBirth(charcodes);
         attributes.values[5] = AttributeLibrary.getGender(charcodes);
         attributes.values[6] = AttributeLibrary.getExpiryDate(charcodes);
-        attributes.values[7] = AttributeLibrary.getOlderThan(charcodes);
-
-        sbtExpiration[newTokenId] = block.timestamp + 90 days;
-    }
-
-    function fieldElementsToBytes(
-        uint256[3] memory publicSignals
-    ) public pure returns (bytes memory) {
-        uint8[3] memory bytesCount = [31, 31, 28];
-        bytes memory bytesArray = new bytes(90); // 31 + 31 + 28
-
-        uint256 index = 0;
-        for (uint256 i = 0; i < 3; i++) {
-            uint256 element = publicSignals[i];
-            for (uint8 j = 0; j < bytesCount[i]; j++) {
-                bytesArray[index++] = bytes1(uint8(element & 0xFF));
-                element = element >> 8;
-            }
-        }
-
-        return bytesArray;
-    }
 
-    function sliceFirstThree(
-        uint256[12] memory input
-    ) public pure returns (uint256[3] memory) {
-        uint256[3] memory sliced;
+        uint[] memory olderThanAscii = new uint[](2);
+        olderThanAscii[0] = p_proof.pubSignals[PROVE_RSA_OLDER_THAN_INDEX];
+        olderThanAscii[1] = p_proof.pubSignals[PROVE_RSA_OLDER_THAN_INDEX + 1];
+        attributes.values[7] = _convertUintArrayToString(olderThanAscii);
```

Comment:
olderThan is now not in revealedData_packed, it is in older_than data field, so I changed this like this